corehq/apps/style/static/style/ko/knockout_bindings.ko.js

```diff
@@ -687,3 +687,13 @@ ko.bindingHandlers.popover = {
         $(element).popover(options);
     }
 };
+
+ko.bindingHandlers.initializeValue = {
+    init: function(element, valueAccessor) {
+        valueAccessor()(element.getAttribute('value'));
+    },
+    update: function(element, valueAccessor) {
+        var value = valueAccessor();
+        element.setAttribute('value', ko.utils.unwrapObservable(value));
+    }
```

Comment:
why was this necessary? it looks fine but it seems like it would affect every knockout model we have, just trying to figure out what it changes